packages/teleport/src/DesktopSession/DesktopSession.tsx

```diff
@@ -56,7 +57,7 @@ export function DesktopSession(props: State) {
   if (fetchAttempt.status === 'failed') {
     alertText = fetchAttempt.statusText;
   } else if (tdpConnection.status === 'failed') {
-    alertText = tdpConnection.status;
+    alertText = tdpConnection.statusText;
```

Comment:
it looks like `statusText` can be optional. Did you want to provide a default value in the event that it's undefined?